k8s-api/src/main/java/io/enmasse/k8s/api/ConfigMapAddressApi.java

```diff
@@ -151,21 +156,18 @@ private ConfigMap create(Address address) {
                 .addToLabels(LabelKeys.INFRA_UUID, infraUuid)
                 .addToLabels(LabelKeys.INFRA_TYPE, "any")
                 .addToAnnotations(address.getAnnotations())
+// I think that resourceVersion is always assigned by the server, so I think doing this unconditionally is safe.
+                .withResourceVersion(address.getResourceVersion())
```

Comment:
I'm wondering if this will work in the initial create via API server, which also calls this create method, and will cause a .withResourceVersion(null) in that case.

Reply:
Looking at the implementation, it seemed to have no affect, but maybe it was better as it was.  I'm inclined to do the same to io.enmasse.k8s.api.ConfigMapAddressSpaceApi#create.